src/script.js

```diff
@@ -0,0 +1,285 @@
+/* eslint-disable no-param-reassign */
+// eslint-disable-next-line import/extensions
+import { keyMap, keys } from './keys.js';
+
+window.onkeydown = (e) => e.preventDefault();
+
+let language = 'en';
+const valueOrShiftValue = 'value';
+let keyboardKeys;
+let capsLkState = false;
+let shiftState = false;
+
+const textarea = document.createElement('textarea');
+const mainSection = document.createElement('div');
+mainSection.classList.add('mainSection');
+const keyboardSection = document.createElement('div');
+keyboardSection.classList.add('keyboardSection');
+const legend = document.createElement('div');
+legend.classList.add('legend');
+legend.insertAdjacentHTML('beforeend', '<p>The Virtual keyboard was created in <b>Windows</b><br>Language switch hotkey - <b>Shift + Alt</b> or <b>Win</b></p>');
+
+document.body.appendChild(mainSection);
+mainSection.appendChild(textarea);
+mainSection.appendChild(keyboardSection);
+mainSection.appendChild(legend);
+
+const saveLangState = () => {
+  sessionStorage.setItem('keyboard-language', language);
+};
+
+const restoreLangState = () => {
+  language = sessionStorage.getItem('keyboard-language') ? sessionStorage.getItem('keyboard-language') : 'en';
+};
+
+restoreLangState();
```

Comment:
restoration is better to do on window.onload event

Reply:
Fixed it. Thanks.